python/lsst/analysis/tools/atools/photometricRepeatability.py

```diff
@@ -193,5 +199,21 @@ def setDefaults(self):
         self.process.buildActions.statMask.threshold = 200
         self.process.buildActions.statMask.fluxType = "psfFlux"
 
+        self.process.calculateActions.photResidFocalPlaneMedian = MedianAction(vectorKey="z")
+        self.process.calculateActions.photResidFocalPlaneStdev = StdevAction(vectorKey="z")
+        self.process.calculateActions.photResidFocalPlaneSigmaMad = SigmaMadAction(vectorKey="z")
+
         self.produce.plot = FocalPlanePlot()
         self.produce.plot.zAxisLabel = "Mag - Mag$_{mean}$ (mmag)"
+
+        self.produce.metric.units = {  # type: ignore
+            "photResidFocalPlaneSigmaMad": "mmag",
+            "photResidFocalPlaneStdev": "mmag",
+            "photResidFocalPlaneMedian": "mmag",
+        }
+
+        self.produce.metric.newNames = {
+            "photResidFocalPlaneSigmaMad": "{band}_photResidFocalPlaneSigmaMad",
+            "photResidFocalPlaneStdev": "{band}_photResidFocalPlaneStdev",
+            "photResidFocalPlaneMedian": "{band}_photResidFocalPlaneMedian",
+        }
```

Comment:
These quantities don't really have any connection with the focal plane, so I don't think you want to have "FocalPlane" in the metric name. Maybe "Tract" instead, since this is calculated on all the sources in a tract?

Reply:
Good point -- only the plotting is actually happening over the focal plane, while the metric is over a tract. I'll change them to "Tract" instead.